action.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Hello World'
+description: 'Greet someone'
+inputs:
+  sources:
+    description: 'One or more directories / files to upload '
+    required: true
+    default: '.'
+  artifacts_url:
+    description: 'artifacts url'
+    required: true
+  username:
+    description: 'artifacts username'
+    required: false
+  password:
+    description: 'artifacts password'
+    required: false
+runs:
+  using: "composite"
+  steps:
+    - id: additional-xargs-args
+      shell: bash
+      run : |
+        if [ $(cat /etc/*-release | grep alpine | wc -l) -eq 0 ]; then
+                echo "::set-output name=args::'-P 16'"
+        fi
+    - id: directory-path
+      shell: bash
+      run: echo "::set-output name=url::github:scality:action-artifacs-upload:dev-staging-1.0.0.0.r210721133720.d674663359.pre-merge.00118602"
```

Comment:
This variable is going to be called artifacts_name (or artifact-name as I'm not sure what is the common naming syntax in action) I recommend you to already name it this way.